### DISCUSSION TITLE
Would be possible to adapt the tool for Chakra UI Pro?

### DISCUSSION THREAD

**Discussion by leetweezard** [kiliman/tailwindui-crawler#82] [Ideas]
Hi there, I've been thinking if would be possible to adapt the tool for Chakra UI Pro or if this is not the case, would you guys be up to roll-out a new tool for it? I could provide help with the Chakra UI Pro subscription for testing as I own the Bundle license (Marketing + Application UI + E-Commerce). Congratulations for the great work!

**Comment by kiliman** [comment]
I imagine it is doable. Perhaps I can generalize the crawler and then create custom implementations to extract the content for each site. I see that they offer a preview version, so I can test the crawler against that first.

**Reply by leetweezard** [reply]
Yes, this is correct. The only problem would be with login as they use passwordless login through Gmail, Github, etc or via email where they send a 4-digit code to complete the login. Do you think we can manage to login through the second option with the 4-digit code?

Moreover, they offer components both in JS and TS and with different styles, this means the script would need 2 new options in the .env file where users can specify if downloading JS, TS or Both (in separate folders) and another option where users can specify which theme(s) to download or if downloading all of them (in separate folders too).

**Comment by leetweezard** [comment]
Hey @kiliman did you go through this by any chance? Let me know how I could be of any help if needed! ;)

**Reply by kiliman** [reply]
Not really. I did do some research on doing OAuth 2 from the command line so should be able to support logins.

**Reply by leetweezard** [reply]
Okay, let me know if I can be of any help.

**Reply by leetweezard** [reply]
Hey, any updates? I was wondering if could make sense also adding support for TailwindUIKit (tuk.dev). If you can provide me with some suggestions and eventually some instructions, I could help to add the support to them! :)
